src/liballoc/rc.rs

```diff
@@ -109,8 +112,8 @@
 //!
 //!     // Despite dropping `gadget_owner`, we're still able to print out the name
 //!     // of the `Owner` of the `Gadget`s. This is because we've only dropped a
-//!     // single `Rc<Owner>`, not the `Owner` it points to. As long as there are
-//!     // other `Rc<Owner>` pointing at the same `Owner`, it will remain
+//!     // single `Rc<Owner>`, not the `Owner` allocation it points to. As long as there are
+//!     // other `Rc<Owner>` pointing at the same `Owner` allocation, it will remain
```

Comment:
The diff above and below notes that `Weak<T>`s also keep the allocation alive.

Reply:
Hm, yeah... but what matters more here is that `Owner` has not been dropped yet.

I adjusted this in 696cba6e25f; better that way?

Reply:
👍